proposals/20230511-roadmap-management.md

```diff
@@ -0,0 +1,93 @@
+# Falco Roadmap Management Proposal
+
+## Summary 
+
+This document proposes introducing a structured process for the management of the roadmap management and related changes in our development process to make effective the execution of our roadmap.
+
+### Goals
+
+The pillars of this proposal are:
+
+- Define processes for release cycles and development iterations
+- Provide guidelines for planning and prioritizing efforts
+- Introducing regular meetings for core maintainers
+- Using *GitHub Project* as the primary tool for managing *The Falco Project* roadmap
+
```

Comment:
Non-goals include providing an exact set of criteria for task prioritization.

Reply:
Would we like to specify also non-goals in this proposal maybe @incertum?
WDYT @leogr?

Reply:
In my experience mentioning a few non-goals is always beneficial to set expectations no matter in what context.

Reply:
Thank you! I've incorporated several non-goals as you suggested :heart: